apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -0,0 +1,253 @@
+                        aria-label="Close navigation menu"
+                        onClick={close}
+                        className="text-white"
+                      >
+                        <SfIconClose />
+                      </SfButton>
+                    </div>
+                    {categoriesContent.map(({ heading, items }) => (
+                      <div key={heading}>
+                        <h2
+                          role="presentation"
+                          className="typography-text-xs font-bold text-neutral-900 tracking-widest whitespace-nowrap px-4 py-2"
+                        >
+                          {heading}
+                        </h2>
+                        <hr className="mb-3.5" />
+                        <ul>
+                          {items.map((item) => (
+                            <SfListItem size="sm" role="none" key={item.title}>
+                              <a
+                                className="focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
+                                href={item.link}
+                              >
+                                {item.title}
+                              </a>
+                            </SfListItem>
+                          ))}
+                        </ul>
+                      </div>
+                    ))}
+                    <div className="flex flex-col items-center justify-center bg-neutral-100 rounded-md border-neutral-300 overflow-hidden grow">
+                      <img src={bannerDetails.image.src} alt={bannerDetails.title} className="object-contain" />
+                      <p className="mb-4 mt-4 px-4 text-center typography-text-base font-medium">
+                        {bannerDetails.title}
+                      </p>
+                    </div>
+                    <SfButton
+                      square
+                      size="sm"
+                      variant="tertiary"
+                      aria-label="Close navigation menu"
+                      onClick={close}
+                      className="hidden md:block md:absolute md:right-0 hover:bg-white active:bg-white"
+                    >
+                      <SfIconClose className="text-neutral-500" />
+                    </SfButton>
+                  </SfDrawer>
+                </CSSTransition>
+              </li>
+            </ul>
+            <div>
+              {actionItems.map((actionItem) => (
+                <SfButton
+                  className="mr-2 -ml-0.5 text-white bg-transparent hover:bg-primary-800 hover:text-white active:bg-primary-900 active:text-white"
+                  key={actionItem.ariaLabel}
+                  aria-label={actionItem.ariaLabel}
+                  variant="tertiary"
+                  slotPrefix={actionItem.icon}
+                  square
+                >
+                  {actionItem.role === 'login' && <p className="hidden md:inline-flex">Log In</p>}
```

Comment:
You added a label to the `actionItem` config, so you can use it here. I'd assume I can configure it there, and it'd get me confused if it didn't change here. Same in the NavbarTop component.
```suggestion
                  {actionItem.role === 'login' && <p className="hidden md:inline-flex">{actionItem.label}</p>}
```